include/kodlab_mjbots_sdk/mjbots_control_loop.h

```diff
@@ -177,7 +181,10 @@ MjbotsControlLoop<log_type, input_type, robot_type>::MjbotsControlLoop(std::vect
 template<class log_type, class input_type, class robot_type>
 MjbotsControlLoop<log_type, input_type, robot_type>::MjbotsControlLoop(std::shared_ptr<robot_type>robot_in, const ControlLoopOptions &options)
   : AbstractRealtimeObject(options.realtime_params.main_rtp, options.realtime_params.can_cpu),
-    lcm_sub_(options.realtime_params.lcm_rtp, options.realtime_params.lcm_cpu, options.input_channel_name) {
+    lcm_sub_(options.realtime_params.lcm_rtp, options.realtime_params.lcm_cpu)
```

Comment:
How many lines before we write a SetupLCMInput()? ;P

Reply:
Perhaps make an issue?

Reply:
I think it fine for now